habitat-hitl/habitat_hitl/_internal/networking/networking_process.py

```diff
@@ -518,8 +518,24 @@ async def http_handler(request):
         # print(f"Returned availability HTTP code {code}")
         return aiohttp.web.Response(status=code)
 
+    async def get_status(request):
```

Comment:
A bit confusing name since the availability server is already using "HTTP status codes" (aka response codes).